client/my-sites/site-settings/form-general.jsx

```diff
@@ -737,7 +737,7 @@ export class SiteSettingsFormGeneral extends Component {
 
 				{ ! siteIsJetpack && (
 					<div className="site-settings__footer-credit-container">
-						<SettingsSectionHeader title={ translate( 'Footer Credit' ) } />
+						<SettingsSectionHeader title={ translate( 'Footer credit' ) } />
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'Footer credit changed' )` *ES Score: 7*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/card-headings-form-labels-sentence-case)